src/views/edit/edit-title-view.vue

```diff
@@ -62,7 +62,7 @@
             v-if="id"
             :label="$t('component.general.uuid.label')"
             :value="titleItem.uuid"
-            path="/title"
+            path="/#/title"
```

Comment:
This does not work for creating an URL which points to the REST API for example. We do not want a link to this client.